### DISCUSSION TITLE
目录细化

### DISCUSSION THREAD

**Discussion by ZhuoZhuoCrayon** [doocs/technical-books#31] [Ideas]
当前情况
* 粗略划分几个大致的方向
* 后续书籍增加后同细化类别书籍难以group

建议：
* 大类拆分，例如`Backend` 下可以有`Java` `Python`等细分项
* 对类别进行描述定义，例如`common` - 计算机基础书籍 等...

**Comment by yanglbme** [comment]
很好的建议
